ckanext/security/helpers.py

```diff
@@ -1,5 +1,40 @@
+from ckan.common import _
 from ckan.plugins.toolkit import asbool, config
+import string, random
```

Comment:
The `random` library is not cryptographically strong. Consider the `secrets` library instead.